js/classes/entities/midground/chad.js

```diff
@@ -179,7 +179,7 @@ class Chad {
         }
         // If Chad can double jump and user is trying to jump than do it!
         if (this.canDoubleJump) {
-            ASSET_MGR.playAudio("./sfx/temp_jump.wav", 0.2);
+            ASSET_MGR.playAudio(SFX.JUMP2.path, SFX.JUMP2.volume);
```

Comment:
Good idea adding volume here like this. Perhaps we can link all of the sfx volume together and make a volume slider for it and make a separate one for music @NathanHinthorne